client/src/pages/product.jsx

```diff
@@ -120,3 +145,42 @@ const ErrorImage = styled.img`
 const ErrorMessage = styled.p`
   padding: 1rem;
 `;
+
+const OtherPicks = styled.ul`
```

Comment:
The naming does not include an hint that this is a list.

Reply:
Let's say you now know what html element the styled component `OtherPicks` is, but believe me, when you have another look at your code in four weeks, you won't. This is why it is important to name everything in a proper manner.

Reply:
I definitely get your point now! ;)